Localization/en-US/covid-19.po

```diff
@@ -0,0 +1,92 @@
+msgstr ""
+
+msgid "An experimental structure of an Ebola protein doesn’t have obvious druggable sites (no deep pockets among the atoms shown as spheres).Our simulations captured a motion that creates a potentially druggable site in this Ebola protein. Instead of showing spheres for each atom, this cartoon shows a ribbon tracing the linear chain of amino acids (chemicals) the protein is made of."
+msgstr ""
+
+msgid "We want to do the same thing with coronavirus."
+msgstr ""
+
+msgid "Spreading the knowledge, Opensource"
+msgstr ""
+
+msgid "When we are done with our analysis we upload the data here:"
+msgstr ""
+
+msgid "We also make all data publicly available, so that other people working in the field can check our analysis and anyone with new methods (e.g. the always growing machine learning data analysis) can look at them at any time: [https://osf.io/2h6p4/wiki/home/](https://osf.io/2h6p4/wiki/home/) and [https://osf.io/dp4cb/wiki/home/](https://osf.io/dp4cb/wiki/home/)"
+msgstr ""
+
+msgid "We are also collaborating with other labs and groups outside the Foldingathome Consortium to solve COVID-19 asap."
+msgstr ""
+
+msgid "What you can do"
+msgstr ""
+
+msgid "1. Donate computing power:"
+msgstr ""
+
+msgid "[Downloading Folding@home](/start-folding/) and helping us run simulations is the primary way to contribute. These calculations are enormous and every little bit helps! Each simulation you run is like buying a lottery ticket. The more tickets we buy, the better our chances of hitting the jackpot. Usually, your computer will never be idle, but we’ve had such an enthusiastic response to our COVID-19 work that you will see some intermittent downtime as we sprint to setup more simulations. Please be patient with us! There is a lot of valuable science to be done, and we’re getting it running as quickly as we can."
+msgstr ""
+
+msgid "Don’t miss our client FAQ. [HERE](https://foldingforum.org/viewtopic.php?f=24&t=32463&fbclid=IwAR1v8Z_2VbB__5izLF989xPRRg9WIYzbd9d5gkfAaP3jCPdkcJbeyUpIt-U)"
+msgstr ""
+
+msgid "To focus on Covid19 research set your category to any."
+msgstr ""
+
+msgid "2) If you don’t have computers to contribute or are feeling particularly generous, you can also [make donations](https://gifts.wustl.edu/med/index.html?other_designation_description=Folding@Home%20(Dr%20Greg%20Bowman,%20Dept%20of%20Biochemistry)&0_d_tr1=71&sc=NG) through Washington University in St. Louis. These funds are used for a number of purposes, including:"
+msgstr ""
+
+msgid "2.1) Supporting our software engineering and server-side hardware (particularly important right now as we scale up rapidly!)"
+msgstr ""
+
+msgid "2.2) Buying compounds to test experimentally based on insight from our simulations."
+msgstr ""
```

Comment:
Should we remove the enumeration from the translation file and keep that only in the template?

Reply:
We shouldn't, because it is going to make template writing more difficult. Different people expect different things when it comes to enumeration.